.github/workflows/create-ubuntu-distribution-package.yml

```diff
@@ -0,0 +1,81 @@
+name: Build & Package (Ubuntu)
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+  pull_request:
+    branches: [ main, master ]
+
+jobs:
+  build-and-package:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        lfs: true
+
+    - name: Setup Git LFS
+      run: |
+        git lfs install
+        git lfs pull
+
+    - name: Install system dependencies
+      run: |
+        sudo apt update
+        sudo apt install -y cmake build-essential clang pkg-config
+
+    - name: Install ICU 77.1
+      run: |
+        cd /tmp
+        wget https://github.com/unicode-org/icu/releases/download/release-77-1/icu4c-77_1-src.tgz
```

Comment:
Why do we download and compile sources? Why not just download and install binary release for Ubuntu? It takes time to compile ICU.

One needs to only copy the $download_location/usr tree to /usr/local/ folder

Reply:
See https://github.com/unicode-org/inflection/pull/128/files for example on how to do it.